solenoid/records/models.py

```diff
@@ -65,26 +65,38 @@ def create_citation(paper_data):
         Some Elements papers include the citation field in their metadata,
         but some leave it blank; in those cases we need to build our own
         citation text. Stakeholders have indicated that it doesn't matter what
-        citation format we use or whether the citation is complete (as long as
-        it includes author, title, and journal title).
-
+        citation format we use or whether the citation is complete.
```

Comment:
Appreciate this small but important change to to docstrings here, that's helpful for understanding it's okay to have missing data in the citation.